@aphecetche : This seems to be a genuine error:
```
/mnt/mesos/sandbox/sandbox/o2suite-dataflow/sw/SOURCES/O2/6458/0/Detectors/Raw/src/RawFileWriter.cxx: In function 'void o2::raw::assertOutputDirectory(std::string_view)':
/mnt/mesos/sandbox/sandbox/o2suite-dataflow/sw/SOURCES/O2/6458/0/Detectors/Raw/src/RawFileWriter.cxx:749:1: error: expected '}' at end of input
```

--- a/Detectors/Raw/src/RawFileWriter.cxx
+++ b/Detectors/Raw/src/RawFileWriter.cxx
@@ -740,10 +740,10 @@ void o2::raw::assertOutputDirectory(std::string_view outDirName)
     if (!std::filesystem::exists(outDirName)) {
       LOG(FATAL) << "could not create output directory " << outDirName;
     }
-  }
 #else
     if (!std::filesystem::create_directories(outDirName)) {
       LOG(FATAL) << "could not create output directory " << outDirName;
     }
 #endif
+  }
 }

